lib/src/Base/Func/TensorizedUniVariateFunctionFactory.cxx

```diff
@@ -45,7 +45,7 @@ typedef ProductUniVariateFunctionEvaluation::UniVariateFunctionCollection UniVar
 
 /* Default constructor */
 TensorizedUniVariateFunctionFactory::TensorizedUniVariateFunctionFactory()
-  : FunctionalBasisImplementation()
+  : BasisImplementation()
   , coll_()
```

Comment:
Is coll_ member still needed here, now that BasisImplementation contains a collection?

Reply:
Yes; coll_ is a collection of UniVariateFunctionFamily:
```typedef Collection<UniVariateFunctionFamily>           FunctionFamilyCollection;```